data/pages/en/privacy_policy.md

```diff
@@ -0,0 +1,29 @@
+---
+title: OCaml Privacy Policy
+description: "<b>TL;DR:</b> We do not use cookies and we do not collect any personal data. We also don't use any third party service."
+meta_title: OCaml Privacy Policy
+meta_description: OCaml.org doesn’t use cookies and doesn’t collect personal data. Your data is your data, period.
+---
+
+The OCaml.org committee is committed to complying with GDPR, CCPA, PECR and other privacy regulations on this website.
+
+## What we collect?
+
+The privacy of your data is very important to us, and so we go to great length to not collect any personal data, and protect you from third party services that may track you.
+
+As a visitor to the OCaml.org website:
+
+- No personal information is collected
+- No information such as cookies is stored in the browser
+- No information is shared with, sent to or sold to third-parties
+- No information is shared with advertising companies
+- No information is mined and harvested for personal and behavioral trends
+- No information is monetized
+
+## Changes and questions
+
+We may update this policy as needed to comply with relevant regulations and reflect any new practices. Whenever we make a significant change to our policies, we will also announce them on our company blog or social media profiles.
+
+Contact us if you have any questions, comments, or concerns about this privacy policy, your data, or your rights with respect to your information.
```

Comment:
Href to discuss.ocaml.org here perhaps. We dont have a email contact for the site.